submissions/VikTorNik/html-css-popup/style.scss

```diff
@@ -0,0 +1,179 @@
+@import "null_style.scss";
+
+body {
+    height: 100%;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 18px;
+}
+
+.wrapper {
+    min-height: 100%;
+    display: flex;
+    flex-direction: column;
+}
+
+.header {
+    background-color: rgb(68, 202, 206);
+}
+
+.nav {
+
+    .nav__menu {
+
+        display: flex;
+        flex-direction: row;
+        justify-content: space-around;
+        height: 50px;
+
+        li {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            width: 100%;
+            font-weight: 500;
+            text-shadow: 3px 3px 8px rgb(7, 68, 64);
+            text-transform: uppercase;
+            color: #000;
+
+            &:focus-within a,
```

Comment:
Avoid using too specific CSS selectors. [Read more about that](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity).